inc/model.class.php

```diff
@@ -376,6 +376,12 @@ function showPartFormUninstall() {
       Dropdown::showYesNo("raz_antivirus",
                           (isset($this->fields["raz_antivirus"])
                            ? $this->fields["raz_antivirus"] : 1));
+      echo "</td>";
+      echo "<td>" . __('Delete inventory data (Lock, Agent, Collect ...)', 'uninstall') . "</td>";
+      echo "<td>";
+      Dropdown::showYesNo("raz_glpiinventory",
+                           (isset($this->fields["raz_glpiinventory"])
+                           ? $this->fields["raz_glpiinventory"] : 0), -1, ['width' => '100%']);
```

Comment:
This is not define for other fields, so I guess it should not be for this one.
```suggestion
                           ? $this->fields["raz_glpiinventory"] : 0), -1);
```